change/@rnw-scripts-promote-release-e733b348-2393-45e5-a8a3-5b374988df71.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It looks like these got applied to some packages they shouldn't. This sometimes happens when origin points to the non-msft fork, and beachball looks in the wrong place. You might need to pass something like "--branch upstream/main" when doing a yarn change.